src/Mod/Draft/draftguitools/gui_edit_arch_objects.py

```diff
@@ -73,12 +73,6 @@ def update_object_from_edit_points(self, obj, node_idx, v, alt_edit_mode=0):
             if vz.Length > 0:
                 obj.Height = vz.Length
 
-    def get_edit_point_context_menu(self, obj, node_idx):
-        pass
-
-    def evaluate_context_menu_action(self, edit_command, obj, node_idx, action):
-        pass
```

Comment:
I would put the removals of all these empty functions in a separate commit, since I think they are orthogonal to generalization changes (I think they can be done either before or after the main commit, iow, the main commit does not require removing these, and removing these doesn't require the main commit, so best to separate into separate commits).

Reply:
I agree that separated commits would have been better, but can I keep the suggestion for the next contribution? :-)

Reply:
Well, ultimately, that's not up to me, but I would think that splitting/restructuring the commits to improve reviewability would be helpful to reduce load on maintainers and make it easier to merge this. Especially now that you're adding commits that amend to original thing, it becomes more difficult to review this commit-by-commit, leaving the only option of reviewing the complete PR as one, which I would not be happy about as a maintainer.

If you're intimidated by this restructuring because git can be a challenge to work with (I'm quite proficient with it and love its power, but it's certainly not easy if you're less experienced) but you a're open to investing the extra time needed to improve the commit structure, I'll be happy to share some tips on how I'd approach this using some git commands to reduce the amount of work needed.

Reply:
ok, let's do it (i feel a bit guilty cause you are loosing a lot of time). I'm using GitKraken because the command line interface is quite scaring to me as a Windows user...

Reply:
> (i feel a bit guilty cause you are loosing a lot of time)

Don't worry, I can guard my own time quite well, and this would be an investment in FreeCAD's future :-p

I'll provide a more in-depth reply in the main conversion thread of this PR, since it's not longer about this particular change.

In the meanwhile, maybe you could resolve the conversations that you have fixed already? Seems only the PR author and people with repo write access can do that :-)

Reply:
Ok! I suggest a dedicated post in a differente section like Open discussion or Developers forum to have a little bit more visibility: Draft/Arch/Bim are not very watched...

Reply:
Oh, I actually meant in the main thread here on github, but maybe it would be interesting to also point to this conversation on the forum, good point.

Reply:
Wow, it was worth the attempt only to know the reset function :-) .
I kept using GitKraken, also if I will definitely try the command line git in the future because I stepped in several workarounds to make it, expecially dragging an dropping files in explorer to edit intermediate commits... 
It would be really nice to have it in the FC wiki IMHO.